### DISCUSSION TITLE
Reconsider changing Freenode to something else

### DISCUSSION THREAD

**Discussion by allendema** [searx/searx#2825] [Ideas]
Consider changing Freenode with something better as the staff has left.

[https://en.wikipedia.org/wiki/Freenode#Ownership_change_and_conflict](https://en.wikipedia.org/wiki/Freenode#Ownership_change_and_conflict)
[https://blog.bofh.it/debian/id_461](https://blog.bofh.it/debian/id_461)

**Comment by unixfox** [comment]
@asciimoo what's your thoughts on this?

**Comment by asciimoo** [comment]
Ohi! We have an IRC channel on IRCNet as well.

**Reply by allendema** [reply]
Maybe replacing Freenode links with IRCNet on /about would be a good start.

Privately, I prefer Matrix.

**Reply by unixfox** [reply]
To be fair I'm also interested into a Matrix room, I follow other communities there, and it's very pleasant to use.

@asciimoo If you could create a matrix room I would be very happy to join it and share it with the other members of the searx community.

**Reply by beedell-roke** [reply]
@unixfox and @allendema, should that be posted as separate topic of discussion? I additionally significantly prefer Matrix.

**Reply by unixfox** [reply]
> @unixfox and @allendema, should that be posted as separate topic of discussion? I additionally significantly prefer Matrix.

You can join the matrix channel of searxng, a popular fork of searx: https://github.com/searxng/searxng#contact
